code/__HELPERS/unsorted.dm

```diff
@@ -953,22 +959,40 @@ var/global/image/busy_indicator_hostile
 			else
 				air_master.tiles_to_update += T2*/
 
-proc/DuplicateObject(obj/original, var/perfectcopy = 0 , var/sameloc = 0)
-	if(!original)
-		return null
 
-	var/obj/O = null
+/proc/DuplicateObject(atom/original, perfectcopy = TRUE, sameloc, atom/newloc)
```

Comment:
There are no instances in either our or /tg/'s code in where this proc is called and either `perfectcopy` or `sameloc` are expected to have any values besides the standard ones, so they can be safely removed.